.gitlab.mk

```diff
@@ -125,9 +125,14 @@ package: deploy_prepare
 	fi;                                                                                                             \
 	echo VERSION=$$VERSION;                                                                                         \
 	PACKPACK_EXTRA_DOCKER_RUN_PARAMS="--network=host ${PACKPACK_EXTRA_DOCKER_RUN_PARAMS}"                           \
-	TARBALL_EXTRA_ARGS="--exclude=*.exe --exclude=*.dll"                                                            \
+	TARBALL_EXTRA_ARGS="--exclude=*.exe --exclude=*.dll --exclude=debian-gc64"                                      \
 	PRESERVE_ENVVARS="PRODUCT_NAME,TARBALL_EXTRA_ARGS,${PRESERVE_ENVVARS}" ./packpack/packpack
 
+package:
+		echo "Creating the package"; \
```

Comment:
Either to remove this line or add the same line to `deploy` target. Otherwise, we will have inconsistency.

Reply:
I'll add the line to deploy target for consistency, because the target is not working without any output

Reply:
done